extractors/marketing-sales/gooddata-reports/index.md

```diff
@@ -41,11 +41,11 @@ Continue by providing your own GoodData **credentials**.
 {: .image-popup}
 ![Screenshot - GoodData Credentials](/extractors/marketing-sales/gooddata-reports/04-credentials.png)
 
-You can also choose an existing GoodData **writer** and use its credentials. The ID of an existing GoodData writer is
-in its configuration URL -- e.g. `https://connection.keboola.com/admin/projects/123/writers/gooddata-writer/WRITER_ID`.
+You can also choose an id of **GoodData Project** provisoned in one of the existing GoodData Writer configurations and use its credentials. The project id(pid) of the provisioned project can be found in a GoodData writer configuration.
```

Comment:
bych sem hodil kde to pid najdes

Reply:
pridal som tam dalsi obrazok z configu writeru, a imho uz by to malo byt jasne
![image](https://user-images.githubusercontent.com/1412120/60653990-67a34580-9e4b-11e9-93c5-1e6dd570153e.png)